property_caching/__init__.py

```diff
@@ -38,12 +45,14 @@ def _get_cache(obj, cache_attr_name=_OBJ_CACHE_ATTR_NAME):
     return getattr(obj, cache_attr_name, {})
 
 
-def _get_property_value(fn, obj, cache_attr_name):
+def _get_property_value(fn, obj, cache_attr_name, only_cache_true_vals=False):
+    result = None
```

Comment:
It wouldn't get to it with that specific None, but I have to initialize the variable to something, because on positive cache hits, the variable doesn't get used, but it's still referenced.